FutureNorthants/northants-website
# Add top services to service landing page

Select up to 6 pages to highlight at the top of a ServiceLandingPage. These can be either ServiceLandingPages or a ServicePage. https://github.com/FutureNorthants/northants-design-system/issues/80

Also added icon selection to ServicePage so the icons can be displayed in the frontend.

## To Test

1. Import the latest config `drush config:import`
2. Update a service landing page with some Top Services (up to 6) and publish
3. Update one of the selected Top Services service pages by selecting an icon from the dropdown, then publish
4. Visit https://core-cms.northants.localhost/graphql and test with the below graphql, ensuring you update the path to the service landing page that was updated. The topServices should be returned in the response.

```graphql
{
  route(path: "") {
    __typename
      node {
        __typename
        ... on ServiceLandingPageNode {
          __typename
          id
          title
          topServices {
            id
            __typename
            servicePages {
              title
              url
              icon
              __typename
            }
        }
      } 	 
    }
  }
}
```
